kubetest/main.go

```diff
@@ -154,6 +156,8 @@ func defineFlags() *options {
 	flag.StringVar(&o.gcpImageProject, "image-project", "", "Project containing node image family, required when --gcp-node-image=CUSTOM")
 	flag.StringVar(&o.gcpNodes, "gcp-nodes", "", "(--provider=gce only) Number of nodes to create.")
 	flag.StringVar(&o.gcpNodeSize, "gcp-node-size", "", "(--provider=gce only) Size of nodes to create (e.g n1-standard-1).")
+	flag.Var(&o.ginkgoParallel, "ginkgo-parallel", fmt.Sprintf("Run Ginkgo tests in parallel, default %d runners. Use --ginkgo-parallel=N to specify an exact count.", defaultGinkgoParallel))
```

Comment:
this change seems unrelated?

Reply:
This is just cleanup, unrelated to my changes. I moved this ginkgoParallel option because it had been out-of-order alphabetically. Of course, there seem to be other options that are out of order below.